src/services/ui/src/pages/welcome/index.tsx

```diff
@@ -176,10 +176,13 @@ export const Welcome = () => {
         <div className="flex justify-around items-center text-xl py-10 px-10 py-4 max-w-screen-xl mx-auto">
           <h4>Do you have questions or need support?</h4>
           <Button asChild>
-            <Link to="/faq">View FAQ</Link>
+            <Link to="/faq" target="myFrameName">
```

Comment:
For code readability and clarity of purpose, I think the `target` name can be something like `faq-tab`. And a good way we can prevent future bugs is to make this a constant value available globally. I believe @13bfrancis has a `consts` module we can use to make a constant value globally available! 😄